driver-core/src/test/java/com/datastax/driver/core/SchemaAgreementTest.java

```diff
@@ -47,7 +47,7 @@ public void should_set_flag_on_non_schema_altering_statement() {
     assertThat(rs.getExecutionInfo().isSchemaInAgreement()).isTrue();
   }
 
-  @Test(groups = "short", priority = 1)
+  @Test(groups = "short")
```

Comment:
The original commit message says:
```
This test should run last as it makes a schema change and then
immediately checks schema agreement.  Subsequent tests that run, such as
should_check_agreement_through_cluster_metadata may observe schemas
don't agree when they run at fail.
```

So, let's say that this test fails - can't this cause failure of other tests?